pkg/merkle/tree.go

```diff
@@ -16,14 +16,16 @@
 package merkle
 
 import (
+	"context"
 	"fmt"
 	"math/big"
-	"math/bits"
 
+	merkleProofService "github.com/galactica-corp/merkle-proof-service/gen/galactica/merkle"
```

Comment:
Don't use capital letters in package names, including aliases.

https://go.dev/blog/package-names

Reply:
You don't event need an alias here, because `merkle` as the name for current package is not available inside of the package itself. You don't have collision, thus you don't need an alias.